modules/main/src/index.ts

```diff
@@ -68,6 +68,7 @@ export {
   DeckRenderer,
   // Logging
   log,
+  assert,
```

Comment:
Prefix internal utilities with `_` so that you wouldn't have to re-export it here. This function is not intended to be used by the users of this library.